src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameControl.xaml.cs

```diff
@@ -13,20 +11,9 @@ namespace Microsoft.CodeAnalysis.InlineRename.UI.SmartRename;
 /// </summary>
 internal sealed partial class SmartRenameControl : UserControl
 {
-    private readonly SmartRenameViewModel _smartRenameViewModel;
-
     public SmartRenameControl(SmartRenameViewModel viewModel)
```

Comment:
Is this class still used somewhere? I thought since we introduced a combo box, this it no longer used

Reply:
It is still used in the XAML